lib/pure/random.nim

```diff
@@ -121,15 +121,19 @@ else:
 proc rotl(x, k: Ui): Ui =
   result = (x shl k) or (x shr (Ui(64) - k))
 
+template randState*(): untyped =
+  ## Makes the default Rand state accessible from other modules.
+  state
+
 proc next*(r: var Rand): uint64 =
   ## Computes a random ``uint64`` number using the given state.
   ##
   ## See also:
-  ## * `rand proc<#rand,Rand,Natural>`_ that returns an integer between zero and
+  ## * `rand proc<#rand,Natural,Rand>`_ that returns an integer between zero and
   ##   a given upper bound
-  ## * `rand proc<#rand,Rand,range[]>`_ that returns a float
-  ## * `rand proc<#rand,Rand,HSlice[T,T]>`_ that accepts a slice
-  ## * `rand proc<#rand,typedesc[T]>`_ that accepts an integer or range type
+  ## * `rand proc<#rand,range[],Rand>`_ that returns a float
+  ## * `rand proc<#rand,HSlice[T,T],Rand>`_ that accepts a slice
```

Comment:
All links to slice don't work. need help.

Reply:
It's probably related to #11795

Reply:
Ok nothing i can do in this pr